pkgs/backy/nautilus.nix

```diff
@@ -3,24 +3,39 @@
   poetry2nix ? pkgs.poetry2nix,
   fetchFromGitHub ? pkgs.fetchFromGitHub,
   lzo ? pkgs.lzo,
+  python310 ? pkgs.python310,
   ...
 }:
-let src = fetchFromGitHub {
-  owner = "flyingcircusio";
-  repo = "backy";
-  # TODO: this points to a branch head that only differs by the Nautilus-specific commits
-  # from what we use for Luminous, but not the latest `main` head as that currently
-  # fails to build in NixOS.
-  rev = "7914012a4e74dfeca597f9dcde237de5ee2f41b0";
-  sha256 = "sha256-EOWqQa//6UhvIIy3LGsFapmK2LMIBIQ90Om4XGOxS2g=";
-};
+let
+# src = ../path/to/checkout;
+  src = fetchFromGitHub {
+    owner = "flyingcircusio";
+    repo = "backy";
+    # TODO: this points to a branch head that only differs by the Nautilus-specific commits
+    # from what we use for Luminous, but not the latest `main` head as that currently
+    # fails to build in NixOS.
+    rev = "7914012a4e74dfeca597f9dcde237de5ee2f41b0";
```

Comment:
Now that the 21.05 poetry2nix build is fixed, I can also rebase the Nautilus changes onto it and see whether this simplifies the packaging here.

Reply:
yes, please.